csrc/host_ir/host_ir.h

```diff
@@ -492,6 +492,43 @@ class ShardByStream : public Expr {
 // wrapper.
 TensorView* shardByStream(TensorView* source, Val* stream_index, Expr* e);
 
+// SymmetricContiguousView takes a sharded TensorView with
```

Comment:
This sharded TensorView has to be contiguous as well, correct?

Reply:
yes, added it in the comment